### ISSUE TITLE
liveProve, readyProbe HTTP endpoints?

### ISSUE THREAD

**Issue by Mistobaan** [GoogleCloudPlatform/stackdriver-zipkin#52]
I am trying to deploy the docker image gcr.io/stackdriver-trace-docker/zipkin-collector inside kubernetes cluster. I was wondering what would be the best endpoint to have the liveProve / readyProbe functionality for a kubernetes pod.

**Comment by Mistobaan** [comment]
I just noticed in the startup logs that `/health` is enabled

**Comment by bogdandrutu** [comment]
What can I do to help?

**Comment by negz** [comment]
Just chiming in - I run the zipkin collector in a Kubernetes cluster and I'm using the `/health` endpoint for liveness and readiness probes without issue. It would be nice if the endpoint were documented.

**Comment by Mistobaan** [comment]
exactly, the endpoint is there but is not documented. I changed the title of the issue.

**Comment by codefromthecrypt** [comment]
so the health endpoint is mentioned here, but yeah not well documented. We also use this for the HEALTHCHECK directive in docker. I'll move this issue to the main repo, noting that there is an emerging https://github.com/openzipkin/zipkin-helm which should master the info on k8s stuff

https://github.com/openzipkin/zipkin/tree/master/zipkin-server#endpoints
